src/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -239,6 +357,87 @@ The `FlutterAppDelegate` performs functions such as:
   (which can only be detected in the AppDelegate) to
   Flutter for scroll-to-top behavior.
 
+### Creating a FlutterAppDelegate subclass
+Creating a subclass of the the `FlutterAppDelegate` in UIKit apps was shown 
+in the [Start a FlutterEngine and FlutterViewController section][]. 
+In a SwiftUI app, you can create a subclass of the 
+`FlutterAppDelegate` that conforms to the `ObservabeObject` protocol as follows:
+
+```swift
+import SwiftUI
+import Flutter
+import FlutterPluginRegistrant
+
+class AppDelegate: FlutterAppDelegate, ObservableObject {
+  let flutterEngine = FlutterEngine(name: "my flutter engine")
+
+  override func application(
+    _ application: UIApplication,
+    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+      // Runs the default Dart entrypoint with a default Flutter route.
```

Comment:
not sure if i understand this comment. seems like a in-depth topic that may not be suitable for an integration guide.

Reply:
This comment was used in the existing Swift example so I was just trying to be consistent. The section near the bottom of this doc talks through the entry points so I think it's fine to leave

Reply:
gotcha. my flutter knowledge is limited, but i guess it's fine if it's already there.